src/constants/directors.js

```diff
@@ -0,0 +1,51 @@
+import nikita from "../assets/avatar-nikita.jpg";
+import aden from "../assets/avatar-aden.jpg"
+import christian from "../assets/avatar-christian.jpg";
+import drake from "../assets/avatar-drake.jpg";
+import cruz from "../assets/avatar-cruz.jpg";
+import griffin from "../assets/avatar-griffin.jpg";
+
+export const directors = [
+  {
+    name: "Nikita Marks",
+    title: "Founder & CEO",
+    quote:
+      "Our mission is to empower talent to achieve their highest aspirations while maintaining the integrity and passion that drive their craft.",
```

Comment:
I added filler text for 4 out of the 6 personas since I could not find their content in the figma file.